### PR TITLE
Gina/hljs styling

### PR DESCRIPTION
**Pull request by ginazampino**
Highlight.js recolor.

To-do:
- Figure out what's causing the neon green bracket styling and if it's possible to change.
- Figure out if it's possible to re-style error(?) highlighting.

### CODE REVIEW

**Comment by ginazampino** [comment]
I'm noticing these black/red highlights. Haven't figured out how to change them yet, but are these syntax
 errors?
 
 
Screenshot_2

**Comment by leastprivilege** [comment]
I don't really like that color scheme - 

I went back to https://highlightjs.org/

for C# i like the following styles

* vs
* default
* github

github also looks OK for JSON..

**Comment by leastprivilege** [comment]
wrt to the JSON - the single dot is indeed a syntax error that we should probably change. But I have seen similar styling for colons...

image

**Comment by ginazampino** [comment]
No problem, I'll switch from mono-blue to something else. Just have to deal with some maintenance stuff at my apartment this morning. Will do it today.

**Comment by ginazampino** [comment]
Updated colors.

**Comment by leastprivilege** [comment]
..Brock doesn't like GitHub ;) Can we try 'vs' ? 

thanks!

**Comment by ginazampino** [comment]
My bad! How is this?

File: IdentityServer/v5/docs/themes/hugo-theme-learn/static/css/atom-one-dark-reasonable.css [hunk 1]
```diff
@@ -12,99 +6,75 @@ pre {
   box-shadow: -1px 2px 5px 1px rgba(0, 0, 0, 0.1);
 }
 
+/*
+
+Visual Studio-like style based on original C# coloring by Jason Diamond <jason@diamond.name>
```
Comment (brockallen) [inline IdentityServer/v5/docs/themes/hugo-theme-learn/static/css/atom-one-dark-reasonable.css:11]: HA -- Jason's an old friend of mine.

File: IdentityServer/v5/docs/themes/hugo-theme-learn/static/css/atom-one-dark-reasonable.css [hunk 2]
```diff
@@ -1,77 +1,80 @@
...
+  background: #fff !important;
+  border-radius: 20px !important;
+  box-shadow: -1px 2px 5px 1px rgba(0, 0, 0, 0.1);
+}
 
-Atom One Dark With support for ReasonML by Gidi Morris, based off work by Daniel Gamage
+/*
 
-Original One Dark Syntax theme from https://github.com/atom/one-dark-syntax
+Visual Studio-like style based on original C# coloring by Jason Diamond <jason@diamond.name>
```
Comment (brockallen) [inline IdentityServer/v5/docs/themes/hugo-theme-learn/static/css/atom-one-dark-reasonable.css:11]: Ha, @jdiamond's an old friend of mine!

**Pull request closed without merging**